cmd/issuer-rest/startcmd/start.go

```diff
@@ -127,6 +131,10 @@ const (
 	requestTokensFlagUsage = "Tokens used for http request " +
 		" Alternatively, this can be set with the following environment variable: " + requestTokensEnvKey
 
+	profilesFilePathFlagName  = "profiles-file-path"
+	profilesFilePathFlagUsage = "Path to file with issuer profiles."
+	profilesFilePathEnvKey    = "ISSUER_PROFILES_FILE_PATH"
```

Comment:
Can we name this as profile mapping file path ? ie, `ISSUER_PROFILES_MAPPING_FILE_PATH`.